### PR TITLE
Fix simple typo in the documentation.

### PR DESCRIPTION
**Pull request by mf-hyunsoo-kang**
---

There are no `self` in constructors.

https://airflow.apache.org/docs/apache-airflow/stable/howto/custom-operator.html#limitations

Before:

```python
class HelloOperator(BaseOperator):
    template_fields = ("field_a", "field_b")

    def __init__(field_a, field_b) -> None:
        self.field_b = field_b
```

After:

```python
class HelloOperator(BaseOperator):
    template_fields = ("field_a", "field_b")

    def __init__(self, field_a, field_b) -> None:
        self.field_b = field_b
```

### CODE REVIEW

File: docs/apache-airflow/howto/custom-operator.rst [hunk 1]
```diff
@@ -283,7 +283,7 @@ templated field:
         class HelloOperator(BaseOperator):
             template_fields = "field_a"
 
-            def __init__(field_a_id) -> None:  # <- should be def __init__(field_a)-> None
+            def __init__(self, field_a_id) -> None:  # <- should be def __init__(self, field_a)-> None
```
Comment (uranusjr) [inline docs/apache-airflow/howto/custom-operator.rst]:
```suggestion
            def __init__(self, field_a_id) -> None:  # <- should be def __init__(self, field_a) -> None
```

while we’re at this

**Review by uranusjr — APPROVED** [review]

**Review by jscheffl — APPROVED** [review]
Will be able to merge after the failure of static checks has been fixed

**Review by vincbeck — APPROVED** [review]

**Comment by jscheffl** [comment]
Still some failures in static checks. Have you tried running `pre-commit` in your workspace or `breeze static-checks`?

**Comment by mf-hyunsoo-kang** [comment]
Sorry, I didn't push the changes.

`field_a` and `field_b` are renamed to `foo` and `bar`, since they have more consistency with other documentations and they have shorter length not to bother formatter(see below).

Before:

```python
def __init__(
                self, field_b, **kwargs
            ) -> None:  # should be def __init__(self, field_a, field_b, **kwargs)
```

After:

```python
def __init__(self, bar, **kwargs) -> None:  # should be def __init__(self, foo, bar, **kwargs)
```

**Pull request merged**